ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-stable-4.11.yaml

```diff
@@ -69,6 +69,19 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-c2s-ipi-disconnected-private-fips-cert-rotation-p2-f1
```

Comment:
For 4.11, i think no need to be f1. Perhaps f6 (or f7) should be fine.

Reply:
Updated with f7.

Reply:
Per suggestion from slack:
```
Xiaoli Tian:
Now we are working on a daily basis to know if the latest nightly build is good enough
Xiaoli Tian:
Before the fix is GA, let's test it daily
```
Change back f1.